IFC4x3/Properties/d/DampingRatio_2PtTTcE0vEpOii8kDqjGba/Documentation.md

```diff
@@ -0,0 +1 @@
+Indicate the damping ratio of the anemometrograph.
```

Comment:
Generalize?
```suggestion
Indicates the damping ratio of the anemometrograph.
```

Reply:
Updated and generalized: https://github.com/bSI-InfraRoom/IFC-Specification/pull/165/commits/1e8b0740cc2e7e000d58ef2f240b7e5a49640ee9